src/srseng/site/settings.py

```diff
@@ -46,6 +46,7 @@
 MORPHODICT_TARGET_LANGUAGE = "eng"
 
 MORPHODICT_SOURCE_LANGUAGE_NAME = "Tsuut’ina"
+MORPHODICT_LANGUAGE_ENDONYM = "Tsuut’ina"
```

Comment:
```suggestion
MORPHODICT_LANGUAGE_ENDONYM = "Tsúūt'ínà"
```

Preferred spelling (with diacritics).